client/app/bundles/Zoon/components/WorkflowCreator.jsx

```diff
@@ -208,6 +224,11 @@ class WorkflowCreator extends React.Component {
               />)}
             </div>
           </F.Column>
+          {this.state.showingModule && (
+            <div className="lightbox">
+              <ModuleCard m={this.state.showingModule} onClick={() => {}} />
```

Comment:
This `onClick` prop looks like a workaround to prevent `ModuleCard` from creating a link.

I propose that we add a boolean `link` prop, default them both to `false/null`, and use either onClick={doStuff} or link={true}. Maybe throw an exception if both are specified.